spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -225,1255 +229,1336 @@
+                :with_notification_phone_configuration,
+                issuer: 'http://localhost:3000',
+                selected_location_details: { name: name },
+                sponsor_id: usps_ipp_sponsor_id,
+              )
+            end
           end
-        end
-        let(:pending_enrollment) { pending_enrollments.first }
-
-        before do
-          enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
-          allow(InPersonEnrollment).to receive(:needs_usps_status_check).
-            and_return(enrollment_records)
-          allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
-        end
-
-        it 'requests the enrollments that need their status checked' do
-          stub_request_passed_proofing_results
+          let(:pending_enrollment) { pending_enrollments.first }
 
-          freeze_time do
-            job.perform(Time.zone.now)
-
-            expect(InPersonEnrollment).to(
-              have_received(:needs_usps_status_check).
-              with(...reprocess_delay_minutes.minutes.ago),
-            )
+          before do
+            enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
+            allow(InPersonEnrollment).to receive(:needs_usps_status_check).
+              and_return(enrollment_records)
+            allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
           end
-        end
 
-        it 'records the last attempted status check regardless of response code and contents' do
-          enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
-          allow(InPersonEnrollment).to receive(:needs_usps_status_check).
-            and_return(enrollment_records)
-          stub_request_proofing_results_with_responses(
-            request_failed_proofing_results_args,
-            request_in_progress_proofing_results_args,
-            request_in_progress_proofing_results_args,
-            request_failed_proofing_results_args,
-          )
-
-          expect(pending_enrollments.pluck(:status_check_attempted_at)).to(
-            all(eq nil),
-            'failed test precondition:
-            pending enrollments must not set status check attempted time',
-          )
+          it 'requests the enrollments that need their status checked' do
+            stub_request_passed_proofing_results
 
-          expect(pending_enrollments.pluck(:status_check_completed_at)).to(
-            all(eq nil),
-            'failed test precondition:
-            pending enrollments must not set status check completed time',
-          )
+            freeze_time do
+              job.perform(Time.zone.now)
 
-          freeze_time do
-            job.perform(Time.zone.now)
+              expect(InPersonEnrollment).to(
+                have_received(:needs_usps_status_check).
+                with(...reprocess_delay_minutes.minutes.ago),
```

Comment:
This syntax is new to me.  Would you be up for explaining it to me when we next synchronously chat, please?  My searches proved fruitless.  Thanks!

Reply:
@eileen-nava  The changes in this file are minimal.  I added a describe block `describe 'In-Person Proofing'` on line 232 so the formatting got weird.  There are a few small changes in the describe section (nested) but really it is the new  describe 'Enhanced In-Person Proofing'  section (that should be at the same level).  I think if you compare red/green blocks here - it should be tab/spacing. Please confirm this because I was surprised at all of the changes.

Reply:
@gina-yamada Ah, gotcha!  I viewed the page again with the "hide whitespace" option enabled for the Diff view, and that made things much clearer.  You're correct, a lot of the changes in this file are tab/spacing.

Reply:
Thanks! I inspected it as well. My new block caused a bunch of items to be over the line length so that is why there appears to be so many changes. 

For existing specs I observe changes related to test setup or line length when I view using the "hide whitespace" option. That made it so much easier to see. I knew about it but forget about it. It makes me feel a lot better about this spec file.